core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -135,6 +128,7 @@ public void onPageScrolled(int position, float positionOffset, int positionOffse
 
     @Override
     public void onPageSelected(int position) {
+        state.updateSelectedPosition(position);
```

Comment:
this is regardless of whetehr we have tabs or not yet - the viewpager has set this item, so we should note it